go/worker/compute/executor/committee/node.go

```diff
@@ -846,14 +1158,48 @@ func (n *Node) worker() {
 	}
 	defer hrtNotifier.Stop()
 
+	// Initialize transaction scheduling algorithm.
+	runtime, err := n.commonNode.Runtime.RegistryDescriptor(n.ctx)
+	if err != nil {
+		n.logger.Error("failed to fetch runtime registry descriptor",
+			"err", err,
+		)
+		return
+	}
+	scheduler, err := scheduling.New(
```

Comment:
Hm, not related to this PR, but what happens if the runtime descriptor is changed? The scheduler will never update its configuration. Maybe we should open an issue for this.

Reply:
oh yeah that's true, currently it wouldn't take affect until the worker is restarted.  i'll open an issue

Reply:
https://github.com/oasisprotocol/oasis-core/issues/3203